oracle/index.rst

```diff
@@ -13,31 +13,22 @@ and features needed to run specific workloads.
    :start-after: Start: Product need and user
    :end-before: End: Product need and user
 
-----------
-
-Canonical's offerings on Oracle Cloud
--------------------------------------
+---------
 
-Canonical builds optimised and certified server images for Oracle Cloud. The images are shared with Oracle privately, who publishes them into the cloud platform once per month. The set of images include:
+In this documentation
+---------------------
 
-* All LTS releases still covered by standard security maintenance (see the `Ubuntu release cycle`_)
-* Full and minimal versions of the images
-* Versions for x86_64 and AArch64 (ARM64) platforms
+..  grid:: 1 1 2 2
```

Comment:
This grid doesn't seem to be rendering correctly in github, not sure if it's meant to

Reply:
It's not meant to. It renders fine in the published version (which you can see if you open the 'details' link for oracle in the list of checks below)